bridges/SamsungMobileChangelogBridge.php

```diff
@@ -0,0 +1,99 @@
+<?PHP
+
+class SamsungMobileChangelogBridge extends BridgeAbstract
+{
+    const NAME = 'Samsung Mobile Changelog';
+    const URI = 'https://doc.samsungmobile.com/';
+    const DESCRIPTION = 'Returns the changelog of selected devices from the Samsung Mobile documentation in English';
+    const MAINTAINER = 'ajain-93';
+    const PARAMETERS = [
+        [
+            'devices' => [
+                'name' => 'Device Model',
+                'required' => true,
+                'exampleValue' => 'SM-S931B',
+            ],
+            'region' => [
+                'name' => 'Region',
```

Comment:
Use the `title` attribute to show a tooltip where you explain what is required here. How can users find out what they have to enter or what valid options there are? (but keep it concise at the same time). Maybe also do the same for the device model.